guides/source/active_record_multiple_databases.md

```diff
@@ -403,8 +403,7 @@ It's also possible to swap connections granularly for shards.
 
 ```ruby
 AnimalsRecord.connected_to(role: :reading, shard: :shard_one) do
-  Dog.first # Will read from shard_one_replica. If no connection exists for
-  shard_one_replica, a ConnectionNotEstablished error will be raised
+  Dog.first # Will read from shard_one_replica. If no connection exists for shard_one_replica, a ConnectionNotEstablished error will be raised
```

Comment:
We try to wrap new docs at 80 characters, so I think this is better:

```suggestion
  Dog.first # Will read from shard_one_replica. If no connection exists for shard_one_replica,
                  # a ConnectionNotEstablished error will be raised
```

Reply:
Thanks @eileencodes 👍🏼